pacemaker.spec.in

```diff
@@ -404,6 +410,10 @@ rm -rf %{buildroot}
 
 %post
 /sbin/chkconfig --add pacemaker || :
+%if %{with cman} && %{cman_native}
+# see https://bugzilla.redhat.com/1116401
+/usr/sbin/ccs_update_schema >/dev/null 2>&1 || :
+%endif
```

Comment:
> They don't expect the act of installing a package to irreversibly alter a file provided by another package.

Unless the file is dynamic by design and there's even not an authoritative/base version packaged with the "owner" of the file, and unless there's an unwritten event-based protocol around it prescribing that the point of its update is a first moment the files contributing new knowledge for it get installed (regardless of source: `cman` itself, `resource-agents`, `fence-agents`, ...). I detailed this in the spec file comment in the batch.